pkg/rtc/rtc_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2021 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package rtc
+
+import (
+	"os"
+	"runtime"
+	"testing"
+
+	"github.com/u-root/u-root/pkg/testutil"
+)
+
+type testSyscalls struct{}
+
+func (tsc testSyscalls) ioctlGetRTCTime(fd int) (*RTCTime, error) {
+	return &RTCTime{}, nil
+}
+
+func (tsc testSyscalls) ioctlSetRTCTime(fd int, time *RTCTime) error {
+	return nil
+}
+
+func OpenMockRTC() (*RTC, error) {
+	var tsc testSyscalls
+	f, err := os.CreateTemp(os.TempDir(), "rtc-*")
+	if err != nil {
+		return nil, err
+	}
+	return &RTC{
+		f,
+		tsc,
+	}, nil
+}
+
+func TestOpenRTC(t *testing.T) {
+	rtc, err := OpenMockRTC()
+	if err != nil {
+		t.Errorf("Expected: nil Got: %v", err)
```

Comment:
Make sure to put the got before the want:

```suggestion
		t.Errorf("OpenRTC() got err %v; want nil", err)
```